mavproxy/source/docs/getting_started/faq.rst

```diff
@@ -15,7 +15,7 @@ Frequently Asked Questions and Issues
 
 #. Which Flight controllers are compatible with MAVProxy?
 
-    The APM branches (`APM:Plane <http://plane.ardupilot.com/>`_, `APM:Copter <http://copter.ardupilot.com/>`_, `APM:Rover <http://rover.ardupilot.com/>`_) are fully compatible. Other Mavlink-based flight controllers (such as the `PX4 <http://px4.io/>`_ stack) should be compatible, but may show all flight data.
+    The APM branches (`APM:Plane <https://ardupilot.org/plane/>`_, `APM:Copter <https://ardupilot.org/copter/>`_, `APM:Rover <https://ardupilot.org/rover/>`_) are fully compatible. Other Mavlink-based flight controllers (such as the `PX4 <https://px4.io/>`_ stack) should be compatible, but may not show all flight data.
```

Comment:
We're really trying to kill off "APM" - could we replace APM:Plane with ArduPlane and "APM" with ArduPilot", please?

Also - that's horrendously misleading calling them "branches" :-)  Perhaps "The ArduPilot vehicle types"?
```suggestion
    The ArduPilot vehicle types (`ArduPlane <https://ardupilot.org/plane/>`_, `ArduCopter <https://ardupilot.org/copter/>`_, `ArduSub <https://ardusub.com/>` and `ArduRover <https://ardupilot.org/rover/>`_) are fully compatible. Other Mavlink-based flight controllers (such as the `PX4 <https://px4.io/>`_ stack) should be compatible, but may not show all flight data.
```

Reply:
Whoops ... fixed :)